internal/evmreader/evmreader.go

```diff
@@ -36,7 +36,7 @@ type EvmReaderRepository interface {
 		inputs []Input,
 		blockNumber uint64,
 		appAddress common.Address,
-	) error
+	) ([]uint64, error)
```

Comment:
Evm Reader Repository Mock needs to be updated to conform to new the new method signature